packages/cli/src/parsers/ip-parser.ts

```diff
@@ -0,0 +1,23 @@
+/********************************************************************************
+ * Copyright (c) 2025 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the W3C Software Notice and
+ * Document License (2015-05-13) which is available at
+ * https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR W3C-20150513
+ ********************************************************************************/
+import { InvalidArgumentError } from "commander";
+
+export function parseIp(value: string, previous: string) {
+    if (!/^([a-z]*|[\d.]*)(:[0-9]{2,5})?$/.test(value)) {
+        throw new InvalidArgumentError("Invalid host:port combo");
+    }
+
+    return value;
+}
```

Comment:
Several comments:
* Parameter previous is never used
* Does it work with ipv6
* do we need the code at all? To me, if a wrong IP is given it should/will fail somewhere else and it is not our job to test valid IPs..

Reply:
We can remove it, but this code is mainly used to parse and validate the inputs for `--inspect` and `--inspect-brk` and it might be useful to fail early rather than brak the nodejs debugger. Nonetheless, I agree with you that the code is not that great as it doesn't cover ipv6... we can remove it for now.